app/views/layouts/application.html.erb

```diff
@@ -62,7 +62,12 @@
 </head>
 <body data-username='<%= @user.try(:username) %>' data-now-unix='<%= Time.current.to_i  %>'>
   <header id="nav">
-    <a id="logo" style="<%= @traffic_style %>" href="/" title="<%= Rails.application.name %> (Current traffic: <%= @traffic_intensity.to_i %>%)"></a>
+    <a id="logo" aria-label="<%= Rails.application.name %>" style="<%= @traffic_style %>" href="/" title="<%= Rails.application.name %> (Current traffic: <%= @traffic_intensity.to_i %>%)">
+      <svg aria-hidden viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Why inline the logo?

Reply:
Could you check that this works with the novelty logos? They're in `app/helpers/traffic_helper.rb`, function `novelty_logo`. The easiest way to check is to edit the date conditionals to just `if true`.

I'm also concerned by adding 900 bytes to every pageload. I'd rather see this moved into a `.svg` in the asserts directory than inline.